packages/core/src/validate.ts

```diff
@@ -196,7 +263,7 @@ const errorInfo: ErrorDescriptions<ValidationError> = {
   },
   'external-library-linking': {
     msg: e => `Linking external libraries like \`${e.name}\` is not yet supported`,
-    hint: `Stick to libraries with internal functions only`,
+    hint: `If you have manually checked that the libraries are upgrade safe, you can skip this check with the \`unsafeAllowLinkedLibraries\` flag`,
```

Comment:
I would like to keep the previous recommendation, because I suspect some people may not be aware that they could be using internal functions.

```suggestion
    hint: `Use libraries with internal functions only, or skip this check with the \`unsafeAllowLinkedLibraries\` flag if you have manually checked that the libraries are upgrade safe`,
```

Reply:
Done.